src/app/course/course.component.html

```diff
@@ -6,6 +6,10 @@
         {{ course?.name }}
     </h1>
 
+    <h3 class="tui-text_h6">
```

Comment:
the tag should match the class. When you are using text_h6 set that tag as <h6> too.